$OpenBSD: patch-src_test_ui_stack-probes_rs,v 1.1 2019/09/29 08:23:17 semarie Exp $
Disable stack-probes test (MAP_STACK terminate the process in a way the testsuite fails).
Index: src/test/ui/stack-probes.rs
--- src/test/ui/stack-probes.rs.orig
+++ src/test/ui/stack-probes.rs
@@ -8,6 +8,7 @@
 // ignore-sparc
 // ignore-sparc64
 // ignore-wasm
+// ignore-openbsd
 // ignore-cloudabi no processes
 // ignore-emscripten no processes
 // ignore-sgx no processes
